### ISSUE TITLE
PresentationFontCach.exe using large amount of CPU

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/wpf#8732]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/PresentationFontCachexe-using-large-amo/10567470)._

---
[severity:I'm unable to use this version] [regression] [worked-in:17.8.2]
The Windows Presentation Foundation Font Cache 3.0.0.0 service (PresentationFontCache.exe) constantly is using a large portion of my CPU. Even when I reboot, it comes back. It happens when I am not even running Visual Studio. Please assist.

---
#### Feedback Bot on 1/23/2024, 09:28 PM: 

(private comment, text removed)

---
### Original Solutions
(no solutions)

**Comment by miloush** [comment]
That service is not related to Visual Studio. It can be disabled in Services.

**Comment by lindexi** [comment]
@miloush Can I know the dotnet core version of wpf will still use this PresentationFontCache Services?

**Comment by miloush** [comment]
@lindexi No. Since .NET Framework 4.0 switched from its own text rendering stack to DWrite, DWrite is in charge of the cache (that service is now called _Windows Font Cache Service_). The _Windows Presentation Foundation Font Cache 3.0.0.0_ service is only started when running WPF 3.5 app or earlier.

**Comment by lindexi** [comment]
So, disable *Windows Presentation Foundation Font Cache 3.0.0.0* service is safe for dotnet core versions of wpf applications.

**Comment by miloush** [comment]
Disabling the WPF cache service has always been safe as far as I remember, it would just have performance impact caused by, well, not having a font cache. By what the report is suggesting, such impact might be worth exchanging their current experience.
